lib/random.gd

```diff
@@ -237,6 +259,70 @@ fi;
 DeclareOperation( "RandomSource", [IsOperation] );
 DeclareOperation( "RandomSource", [IsOperation, IsObject] );
 
+
+##############################################################################
+##
+##  <#GAPDoc Label="RandomSource_develop">
+##  <Subsection Label="Implementing new kinds of random sources">
+##  <Heading>Implementing new kinds of random sources</Heading>
+##
+##  If one wants to implement a new kind of random sources then
+##  the first step is the declaration of a new category <C>C</C>, say,
+##  that implies <Ref Filt="IsRandomSource"/>, analogous to the categories
+##  listed in Section <Ref Subsect="Kinds of Random Sources"/>.
+##  <P/>
+##  Then the following method installations are needed.
+##  <P/>
+##  <Index Key="Init"><C>Init</C> (initialize a random source object)</Index>
+##  <C>InstallMethod( Init, [ C, IsObject ], function( prers, seed_or_state )
+##  ... end );</C>
+##  <P/>
+##  Here <C>prers</C> is an empty component object (which has already the
+##  filter <C>C</C>), and <C>seed_or_state</C> is an integer or a state value
```

Comment:
Why "seed_or_state"? What you call "state value" is what I'd call "seed" (which does not need to be an integer). This also matches the nomenclature in the existing user facing documentation, I think?

Reply:
O.k.